app/src/main/java/com/example/drawingactivity/DrawActivity.kt

```diff
@@ -0,0 +1,63 @@
+package com.example.drawingactivity
+
+import android.os.Bundle
+import android.widget.Button
+import android.widget.Toast
+import androidx.activity.viewModels
+import androidx.appcompat.app.AppCompatActivity
+
+class DrawActivity : AppCompatActivity() {
+
+    private val viewModel: DrawViewModel by viewModels()
+
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContentView(R.layout.activity_main)
+
+        val dialogUtils = DialogUtils(this, viewModel)
```

Comment:
when you change this to an option you won't need to import it, you would just do DialogUtils.functionName(parameters)